ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/srma/editSrmaPage.ts

```diff
@@ -361,6 +361,24 @@ export class EditSrmaPage {
         return this;
     }
 
+    public verifyTextHint()
+    {
+        Logger.Log("Verify Text Hint ");
+        
+        cy.get('#submission-required-hint').should("exist");
+
+
+        return this;
+    }
+    public verifyTextHintNotDisplayed()
```

Comment:
same here

hasNoReasonHintText, so it clearly describes what it is looking for

Reply:
done